rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml

```diff
@@ -42,9 +42,46 @@ maas_target_alias: public0_v4
 maas_monitor_cinder_backup: "{{ cinder_service_backup_program_enabled | default(false) }}"
 
 # MaaS pathing and versions
-maas_release: master
+maas_release: "1.0.0"
 maas_venv: "/openstack/venvs/maas-{{ maas_release }}"
 maas_venv_bin: "{{ maas_venv }}/bin"
 
 # Allow all of the host machiens running maas agents to restart once at the end of a maas playbook.
 maas_restart_independent: false
+
+# Apt artifact repo configuration
```

Comment:
We should probably mark that the content of this file, starting from this line to the end of the file, will be deleted when no artifacts are available. Less black magic.

Reply:
That only takes place for the AIO, not for any other deployment. Personally I don't think we should be exposing that to a file used in production deployments - it may make people think they can/could do that for production if it suits them.

Reply:
Oh yeah. Good.